public/app/config/collections/library/searchTypes.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+    'use strict';
+
+    define(function () {
+        return [
+            {
+                key: '_fulltext',
+                labelText: 'Keyword',
+                glyphicon: 'search'
+            },
+            {
+                key: 'ca_objects.Author',
+                labelText: 'Author',
+                glyphicon: 'user'
+            },
+            {
+                key: 'ca_objects.preferred_labels',
+                labelText: 'Title',
+                glyphicon: 'certificate'
+            },
+            {
+                key: 'ca_occurrences',
```

Comment:
`ca_occurrences.preferred_labels.name`, otherwise false positives get included from indexing of related objects.